appveyor.yml

```diff
@@ -5,12 +5,19 @@ branches:
   only:
     - l10n_master # The branch that Crowdin targets when updates are made on .po files
     - master      # The branch that is published on Nuget.org
+    - l10n-pack   # For testing purpose
+environment:
+  packageVersion: 1.0.0
 init:
   - git config --global core.autocrlf true
+  - ps: $env:buildVersion = "$env:packageVersion.$env:appveyor_build_number"
+  - ps: $env:nugetVersion = "$env:packageVersion-beta-$env:appveyor_build_number"
```

Comment:
Why do we need two values for a build number? one with `beta` and one without.

Reply:
The `$env:buildVersion` will be used by AppVeyor itself, and `$env:nugetVersion` the one will be used in `.nupkg`

To see `$env:buildVersion` in action, please keep your eyes on the top right while AppVeyor is running, the version will be something like `1.0.0.54` instead of `1.0.54`

Reply:
You don't answer the `why` in my question. I don't think this is necessary. What would happen if you didn't call `Update-AppveyorBuild`? IMO it would still have a version, as this is defined in the template :

![image](https://user-images.githubusercontent.com/1165805/64216101-4ae7b600-ce6c-11e9-9246-c2cb6c2e9bda.png)

The only difference is that you add an extra `.0`, but we don't really care. And you really really really want that, I can change the template in the settings.

Reply:
If you change the template it will be YES, nothing but the new environment variable that I introduced `packageVersion` will take care of this, so whenever we change the version it will change both the AppVeyor & NuGet version, two birds with one stone

Hope it's answer the question

Reply:
Or, if you really want to give a meaningful version to appveyor (for some reason I don't get) then we could use the correct one, which is in `env:nugetVersion`

Reply:
In this case both are equivalent, but I differentiate between them because sometimes we may need to add preleases or qualifier to the package name, but this is not the case for the AppVeyor

Reply:
> but this is not the case for the AppVeyor

Right, because we really don't care about the appveyor version, which is why I don't get why you even want to update it.

Reply:
If we don't care I will modify it right now

Reply:
BOOM it's ready